File: backend/src/main/java/com/ssafy/ganhoho/domain/auth/dto/LoginRequest.java
package com.ssafy.ganhoho.domain.auth.dto;

import lombok.*;

@Getter
@Builder
@AllArgsConstructor(access = AccessLevel.PROTECTED)
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class LoginRequest {
    private String loginId;
    private String password;
    private String fcmToken;
    private int deviceType;
}

File: ganhoho_app/app/src/main/java/com/ssafy/ganhoho/data/model/dto/auth/LoginRequest.kt
package com.ssafy.ganhoho.data.model.dto.auth

// 로그인 요청
data class LoginRequest(
    val loginId: String,   // 로그인 ID
    val password: String,  // 비밀번호
    val fcmToken: String? = "", // FCM 토큰 (선택 사항)
    val deviceType: Int = 0   // 기기 타입
)